Firmware/USMArduinoFirmware/USMArduinoFirmware.ino

```diff
@@ -159,7 +173,14 @@ SSD1306AsciiWire oled;
 void setup()
 {
   // Start the I2C bus
+  #if defined (ESP32)
+  #define I2C_SDA   14    // to do : need this for my crippled ESP32
```

Comment:
Can we remove this?

Reply:
As pointed out in the comment this is a to do. I needed that for testing the ESP32 config on the only ESP32 that I have on hand with limited pins broken out and the standard I2C pins are not included. I didn't finished thinking if the I2C pin configuration option could be usefull and if yes in the config.h of course.
But I understood that the WiFi cofigurations don't belong to the desired path forward?

Reply:
We could put the I2C config in `config.h` I guess, but I think if someone needs that they will just need to make their own changes locally.

Reply:
I suggest that we delete the WiFi branch or just rename it to something with wifi in it's name. Then it could stay there as a reference for those who might be interested in a wifi supporting version.